lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/CompatibilitySpec.hs

```diff
@@ -47,7 +56,22 @@ import qualified Data.ByteString.Char8 as B8
 import qualified Data.Text as T
 
 spec :: Spec
-spec = describe "EncodeAddress & DecodeAddress" $ do
+spec = do
+    addrSpec
+    describe "txId @(Jormungandr n)" $ do
+        it "(txId largeTx) should match golden" $ do
```

Comment:
Nice!
Would be nice to have some more actually covering various cases:

- Transaction with a single input / output
- Old and new addresses

What do you think ?

Reply:
👍

Reply:
Not doing old addresses now as per thread:

https://input-output-rnd.slack.com/archives/GBT05825V/p1560938440087200?thread_ts=1560934314.081600&cid=GBT05825V